src/extensions/extension-discovery.ts

```diff
@@ -327,7 +331,12 @@ export class ExtensionDiscovery extends Singleton {
         isEnabled
       };
     } catch (error) {
-      logger.error(`${logModule}: can't load extension manifest at ${manifestPath}: ${error}`);
+      if (error.code === "ENOTDIR") {
+        // ignore this error, probably from .DS_Store file
+        logger.debug(`${logModule}: failed to load extension manifest through a not-dir-like at ${manifestPath}`);
```

Comment:
Does this prevent the popup dialog error message on startup?

Reply:
No because what ever is causing that couldn't have come from here. The error was always being logged. I just now change what log level it is.

Reply:
Does this PR address an issue?

Reply:
No just a mild annoyance ever since bundled extensions are no longer being filtered.